.github/workflows/build-and-test.yml

```diff
@@ -173,6 +189,10 @@ jobs:
         run: |
           make gotidy
           git diff --exit-code || (echo 'go.mod/go.sum deps changes detected, please run "make gotidy" and commit the changes in this PR.' && exit 1)
+      - name: Check for go vulnerabilities
+        run: |
+          make govulncheck
+          git diff --exit-code || (echo 'Some vulnerable dependencies are in use, this most likely means you need to updated stuff in go.mod/go.sum and/or bump the Go compiler version itself.' && exit 1)
```

Comment:
Why are changes to this file needed, given that we have a `govulncheck` workflow now?

Reply:
As I wrote in the comment above, this PR contains 3 solutions among which we should choose the one we'd like to go with :)

Reply:
Oh, ok. I missed that 👍